router/dealer.go

```diff
@@ -1487,12 +1494,12 @@ func (d *dealer) regCountCallees(msg *wamp.Invocation) wamp.Message {
 	}
 }
 
-func (d *dealer) trySend(sess *wamp.Session, msg wamp.Message) bool {
-	if err := sess.TrySend(msg); err != nil {
-		d.log.Printf("!!! Dropped %s to session %s: %s", msg.MessageType(), sess, err)
-		return false
+func (d *dealer) trySend(sess *wamp.Session, msg wamp.Message) {
```

Comment:
So we dropped the bool indicator of the result... Was it used somewhere somehow? Do now we miss something because of it?

Reply:
Yeah, I saw previously
```
_ = callee.TrySend(&abortMsg)
```

But maybe there are other places which make use of it

Reply:
Yes, it was used in a few rare places. In those places, a `select` is now used. One such example is [here](https://github.com/gammazero/nexus/pull/302/files#diff-7aa5dc1d38bb9c17ccc27538fd6e899b936997d99c091c2b37578e5d5e5b0d9aR994).